src/test/regress/expected/multi_metadata_attributes.out

```diff
@@ -1,15 +1,14 @@
 -- if the output of following query changes, we might need to change
 -- some heap_getattr() calls to heap_deform_tuple(). This errors out in
--- postgres versions before 11.
+-- postgres versions before 11. If this test fails check out
+-- https://github.com/citusdata/citus/pull/2464 for an explanation of what to
+-- do. Once you used the new code for the table you can add it to the NOT IN
+-- part of the query so new changes to it won't affect this test.
 SELECT attrelid::regclass, attname, atthasmissing, attmissingval
 FROM pg_attribute
-WHERE atthasmissing
+WHERE atthasmissing AND attrelid NOT IN ('pg_dist_node'::regclass)
```

Comment:
I have a though time understanding if we can safely do this.
The query got added for pg11 and new non-null columns to our catalogs.
I guess our new non-null value for `isdatanode` changed the output of this test.

We are now not only hiding that column, but any future changes to `pg_dist_node`. Is that allowed because we have already added all required deform calls for the use of `pg_dist_node` in #2464 and therefore don't need to see any changes to this table anymore?

Reply:
As far as I understood the answer to your last question is yes. i.e. this test only exists to notify us when we add a column to tables that we haven't added the deform calls to yet. @pykello is that indeed the case?

Reply:
This should be safe since the only place in the code that should convert a tuple to a worker node is `TupleToWorkerNode` and other places should just read the cached version. Can you please double check that there is no other place that reads from `pg_dist_node` tuples before merging?

Reply:
There's no other place that reads from pg_dist_node